packages/connectivity/connectivity_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.1.0-nullsafety
```

Comment:
should this be `2.0.0-nullsafety` ?

Reply:
I saw that other platform_interfaces are only bumping minor version. Are we going to break platform_interface as well?